controllers/operatorpolicy_status.go

```diff
@@ -373,6 +424,63 @@ var opGroupTooManyCond = metav1.Condition{
 	Message: "there is more than one OperatorGroup in the namespace",
 }
 
+// buildCSVCond takes a csv and returns a shortened version of its most recent Condition
+func buildCSVCond(csv *operatorv1alpha1.ClusterServiceVersion) metav1.Condition {
+	status := metav1.ConditionFalse
+	if csv.Status.Phase == operatorv1alpha1.CSVPhaseSucceeded {
+		status = metav1.ConditionTrue
+	}
+
+	return metav1.Condition{
+		Type:    condType(csv.Kind),
+		Status:  status,
+		Reason:  string(csv.Status.Reason),
+		Message: "ClusterServiceVersion - " + csv.Status.Message,
+	}
+}
+
+func buildDeploymentCond(
+	depsExist bool,
+	unavailable bool,
+	unavailableDeps []appsv1.Deployment,
+) metav1.Condition {
+	status := metav1.ConditionTrue
+	reason := "DeploymentsAvailable"
+	message := "All operator Deployments have their minimum availability"
+
+	if !depsExist {
+		reason = "NoExistingDeployments"
+		message = "No existing operator Deployments"
+	}
+
+	if unavailable {
```

Comment:
Nit: it seems like this boolean could be replaced with `len(unavailableDeps) != 0`